.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -23,6 +24,7 @@ _Due: <2-business-days-before-release>_
 - [ ] Add the Durable Functions package to the [ADO test feed](https://dev.azure.com/durabletaskframework/Durable%20Task%20Framework%20CI/_artifacts/feed/durabletask-test).
 - [ ] Check for package size, make sure it's not surprisingly heavier than a previous release.
 - [ ] Merge (**choose create a merge commit, NOT squash merge**) dev into main. Person performing validation must approve PR.
+- [ ] Keep branch `v3.x` updated with branch `dev`. Do not merge PRs that are sepecific to Durable Functions v2.
```

Comment:
Does this mean we need to cherry-pick commits? I'm not sure I understand the instructions I should follow for "Do not merge PRs that are specific to Durable Functions v2".

Reply:
I think so. I add this because I am worried that some changes only apply to DF v2, not DF v3. If so, we will not merge these PRs into v3.x . That's also why I add this check option in the PR template https://github.com/Azure/azure-functions-durable-extension/pull/2593.

Reply:
That makes sense. I'm just wondering about the process of how to track the PRs that are specific to v2. With these changes, is it expected, during the release, to go through the PRs and see if they have "My changes **should** be added to v3.x branch." checked and then merge those changes into v3.x?

Reply:
I think so. That's the only way I think of now.

Reply:
@bachuv, @nytian: In 99% of cases, I actually don't think we'll need to cherry pick anything. In general, it should suffice to just **pull** the latest changes on V2 to V3 and to manually fix any merge conflicts that arise, if any. That's how Will has been keeping his branches up to date with our changes.  But yes, if we ever find ourselves in a situation where we have to merge a change that's exclusive to V2, then we'll need to start cherry-picking. I doubt that will happen though.

This is more of a reason to release V3 sooner, so that the v3 branch can become the new dev, and we no longer need to keep it up to date!

Reply:
Ok I see, "Do not merge PRs that are specific to Durable Functions v2." is just for any rare cases then. Thanks for clarifying, @davidmrdavid.